### DISCUSSION TITLE
Magento 2 app Error during npm start

### DISCUSSION THREAD

**Discussion by MihaiRast** [scandipwa/scandipwa#2598] [Install & Upgrade]
After running "npx create-magento-app my-app" when I run "npm start" I came across this error:

`> magento-scripts start

❯ Starting project
  ✔ Creating cache folder
  ❯ Checking requirements
    ❯ Running on linux 3.10.0-1062.12.1.el7.x86_64
      ✖ Cannot read property '1' of null
    ◼ Checking phpbrew
    ◼ Checking docker
    ◼ Checking composer environmental variables
    ◼ Checking Node.js
  ◼ Getting cached ports
  ◼ Stopping Docker services
  ◼ Stopping php-fpm
  ◼ Settings project prefix
  ◼ Getting available ports
  ◼ Saving configuration
  ◼ Install PHP
  ◼ Install Composer, prepare FS & download images
  ◼ Configuring PHP extensions
  ◼ Installing Prestissimo
  ◼ Installing Magento
  ◼ Starting docker services
  ◼ Connecting to MySQL server
  ◼ Setting up Magento
  ◼ Starting php-fpm
  ◼ Opening browser
ERROR! Cannot read property '1' of null
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! arelux@0.0.0 start: `magento-scripts start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the arelux@0.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional log`

Please advice.

Note: on log I have this content:

0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'start' ]
2 info using npm@6.14.12
3 info using node@v14.16.1
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle arelux@0.0.0~prestart: arelux@0.0.0
6 info lifecycle arelux@0.0.0~start: arelux@0.0.0
7 verbose lifecycle arelux@0.0.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle arelux@0.0.0~start: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/arelux/public_html/arelux/node_modules/.bin:/usr/local/cpanel/3rdparty/lib/path-bin:/usr/local/cpanel/3rdparty/lib/path-bin:/usr/local/cpanel/3rdparty/lib/path-bin:/usr/local/sbin:/usr/local/b$
9 verbose lifecycle arelux@0.0.0~start: CWD: /home/arelux/public_html/arelux
10 silly lifecycle arelux@0.0.0~start: Args: [ '-c', 'magento-scripts start' ]
11 silly lifecycle arelux@0.0.0~start: Returned: code: 1  signal: null
12 info lifecycle arelux@0.0.0~start: Failed to exec start script
13 verbose stack Error: arelux@0.0.0 start: `magento-scripts start`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1048:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid arelux@0.0.0
15 verbose cwd /home/arelux/public_html/arelux
16 verbose Linux 3.10.0-1062.12.1.el7.x86_64
17 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
18 verbose node v14.16.1
19 verbose npm  v6.14.12
20 error code ELIFECYCLE
21 error errno 1
22 error arelux@0.0.0 start: `magento-scripts start`
22 error Exit status 1
23 error Failed at the arelux@0.0.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

**Comment by ejnshtein** [comment]
What system are you running?

**Reply by MihaiRast** [reply]
Running on linux 3.10.0-1062.12.1.el7.x86_64 Centos

**Reply by ejnshtein** [reply]
What `phpbrew --version` command prints for you?

**Reply by ejnshtein** [reply]
Because it looks like you have not installed phpbrew correctly

**Reply by MihaiRast** [reply]
@ejnshtein the output of 'phpbrew --version' is:

phpbrew - 1.27.0
cliframework core: 2.5.4

**Reply by ejnshtein** [reply]
I see...
Okay, I'll check what I can do with it.

**Reply by ejnshtein** [reply]
Okay, I made an alpha version that contains some fixes for issues that might appear on your system before. Please try it (`npm i @scandipwa/magento-scripts@alpha`) and say how it is! Thanks!

**Reply by gtkacz** [reply]
Getting the same problem with same _phpbrew_ and _cliframework core_ versions

**Reply by ejnshtein** [reply]
?
